### PR TITLE
874252-UG_documents_volume1_features

### PR DESCRIPTION
**Pull request by SivaranjithN**
Added UG documents for below features in mentioned controls.
### Autocomplete
- ShowBorder
- HorizontalTextAlignment
- VerticaltextAlignment

### ComboBox
- ShowBorder
- HorizontalTextAlignment
- VerticalTextAlignment

### MaskedEntry
- Stroke
- ShowBorder
- HorizontalTextAlignment
- VerticalTextAlignment

### NumericEntry
- ShowBorder
- HorizontalTextAlignment
- VerticalTextAlignment

### Button
- RTL support

### TextInputLayout
- PasswordVisibilityToggled event

### Chip
- IsSelected

### CODE REVIEW

File: MAUI/Chips/Customization.md [hunk 1]
```diff
@@ -1806,4 +1806,6 @@ using Syncfusion.Maui.Core;
 
 N> The default value of ChipImageSize is [`18d`].
 
+## IsSelected
 
+The `IsSelected` property is a public property available in the [`SfChip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfChip.html). This property indicates whether a chip has been selected in choice and filter types.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Chips/Customization.md:1811]: The `IsSelected` property in the [`SfChip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfChip.html) is used to indicate whether a chip has been selected in choice and filter types.

File: MAUI/TextInputLayout/Events.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Events in MAUI TextInputLayout
+
+## PasswordVisibilityToggled Event
+
+The `PasswordVisibilityToggled` event will be triggered whenever toggle the EnablePasswordVisibilityToggle icon in the SfTextInputLayout. It can defines visibility of the password based on the icon clicked. If the icon shows, the value of `IsPasswordVisible` will be **true**. The event arguments are of type `PasswordVisibilityToggledEventArgs` and expose the following property:
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TextInputLayout/Events.md:14]: PLease change the content: The PasswordVisibilityToggled event will be triggered whenever you toggle the password toggle icon in the SfTextInputLayout. The event arguments are of type PasswordVisibilityToggledEventArgs and expose the following property:

File: MAUI/TextInputLayout/Events.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+* `IsPasswordVisible`: Its value is defined based on the password visibility.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TextInputLayout/Events.md:16]: IsPasswordVisible: Its value is defined based on the visibility of the password.

File: MAUI/TextInputLayout/Events.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+inputLayout.Hint = "Password";
+inputLayout.PasswordVisibilityToggled += InputLayout_PasswordVisibilityToggled;
+inputLayout.Content = new Entry() { Text = "1234" }; 
+
+{% endhighlight %}
+
+{% tabs %}
+{% highlight c# %}
+    
+    private void SfTextInputLayout_PasswordVisibilityToggled(object sender, PasswordVisibilityToggledEventArgs e)
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TextInputLayout/Events.md:40]: change method name to OnPasswordVisibilityToggled

File: MAUI/Autocomplete/UI-Customization.md [hunk 5]
```diff
@@ -859,3 +859,59 @@ autocomplete.CursorPosition = 4;
 {% endtabs %}
 
 N> Two way binding not supported in Android platform.
+
+## Border visibility
+
+The [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) ShowBorder property is used to modify the visibility of border and its default value is **true**. The following code example demonstrates how to change the border visibility,
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/UI-Customization.md:865]: The ShowBorder property of  [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) is used to modify the visibility of the border and its default value is **true**. The following code example demonstrates how to change the border visibility,

File: MAUI/Chips/Customization.md (hunk 1 above)
Comment (SivaranjithN) [reply MAUI/Chips/Customization.md:1811]: Addressed the comments.

File: MAUI/TextInputLayout/Events.md (hunk 2 above)
Comment (SivaranjithN) [reply MAUI/TextInputLayout/Events.md:14]: Addressed the comments.

File: MAUI/TextInputLayout/Events.md (hunk 3 above)
Comment (SivaranjithN) [reply MAUI/TextInputLayout/Events.md:16]: Addressed the comments.

File: MAUI/TextInputLayout/Events.md (hunk 4 above)
Comment (SivaranjithN) [reply MAUI/TextInputLayout/Events.md:40]: Addressed the comments.

File: MAUI/Autocomplete/UI-Customization.md (hunk 5 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/UI-Customization.md:865]: Addressed the comments.

File: MAUI/TextInputLayout/Events.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Events in MAUI TextInputLayout control | Syncfusion
+description: Learn about Events support in Syncfusion Essential Studio MAUI TextInputLayout control, its elements, and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Events in MAUI TextInputLayout
```
Comment (Choza-rajan) [inline MAUI/TextInputLayout/Events.md:10]: Use Events

Comment (SivaranjithN) [reply MAUI/TextInputLayout/Events.md:10]: I have added the title.

File: MAUI/Autocomplete/UI-Customization.md [hunk 7]
```diff
@@ -859,3 +859,61 @@ autocomplete.CursorPosition = 4;
...
+
+The following image illustrates the result of the above code:
+
+![ShowBorder](Images/UICustomization/ShowBorder.png)
+
+### TextAlignment
+
+The [`SfAutocomplete`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) provides support to customize the text alignment by using the `HorizontalTextAlignment` and `VerticalTextAlignment` properties.
+
+N> `HorizontalTextAlignment` dynamic changes is not working in Android platforms.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/UI-Customization.md:895]: Dynamic changes to the HorizontalTextAlignment property may not be functioning as expected on Android platforms.

Comment (SivaranjithN) [reply MAUI/Autocomplete/UI-Customization.md]: Addressed the comments.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**